run-queue

```diff
@@ -142,49 +143,71 @@ Subject: cockpituous: run-queue crash on %s
 # Consume from the webhook queue and republish to the task queue via *-scan
 # Consume from the task queue (endpoint)
 def main():
-    parser = argparse.ArgumentParser(description='Bot: read a single test command from the queue and execute it')
+    parser = argparse.ArgumentParser(description='Bot: read and execute task command(s) from the queue')
     parser.add_argument('--amqp', default=distributed_queue.DEFAULT_AMQP_SERVER,
                         help='The host:port of the AMQP server to consume from (default: %(default)s)')
+    parser.add_argument('--loop', action='store_true',
+                        help='Run forever until SIGTERM; by default, run a single task or exit '
+                             'immediately if queue is empty')
+    # this exists mostly to speed up integration tests; for production, 1 min is fine
+    parser.add_argument('--loop-sleep', type=int, default=60, metavar='SECONDS',
```

Comment:
Please drop this argument.

Reply:
Are you ok with a `LOOP_SLEEP` env var? One minute sucks for testing (both interactively and also in the unit test)